Makefile

```diff
@@ -1,3 +1,10 @@
+TARGET_SYS?=arm-linux-gnueabihf
+SYS:=$(shell $(CXX) -dumpmachine)
+ifneq ($(SYS),$(TARGET_SYS))
+	CXX:=$(TARGET_SYS)-c++
```

Comment:
Wouldn't this be working better with `$(TARGET_SYS)-g++` ? On my debian installation, I find a `arm-linux-gnueabihf-g++`, but not `arm-linux-gnueabihf-c++` for instance.